esp/src/eclwatch/css/hpcc.css

```diff
@@ -1488,57 +1533,50 @@ margin-left:-20px;
 }
 
 .dojoxColorPickerOptional input {
-    border:1px solid #a7a7a7;
-    width:25px;
-    padding:1px 3px 1px 3px;
-    line-height:1.1em;
+    border: 1px solid #a7a7a7;
+    width: 25px;
+    padding: 1px 3px 1px 3px;
+    line-height: 1.1em;
 }
 
 .dojoxColorPickerHex input {
-    width:55px;    
+    width: 55px;
 }
 
 .dojoxUpgradeBar {
-    position:absolute;
-    left:0;
-    top:0;
-    width:100%;
-    height:32px;
-    overflow:hidden;
-    z-index:100;
-    background:#f3f2af;
-    box-shadow:0 2px 6px #444;
-    -webkit-box-shadow:0 1px 6px #444;
-    -moz-box-shadow:0 1px 6px #444;
-    font-size:.8em;
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    overflow: hidden;
+    z-index: 100;
+    background: #f3f2af;
+    box-shadow: 0 2px 6px #444;
+    -webkit-box-shadow: 0 1px 6px #444;
+    -moz-box-shadow: 0 1px 6px #444;
 }
 
```

Comment:
Just FYI, these couple or three deletions right around here, related to the .dojoxUpgradeBar selector, were the only actual changes I'd made. The rest of this was due to code style rules applied when saving the file in VS Code.